esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -381,6 +382,17 @@ struct CWsWuZAPInfoReq
     {
         logFilter.populateLogFilter(wuid.str(), httpRequest);
     }
+
+    // True when logs _would have been included_ if the user had access
+    bool logsExcludedDueToNoAccess()
```

Comment:
minor, but for some reason the current method name misleads me into thinking the logs have already been excluded, but I think the method is determining if the logs should be excluded,  does it make sense to rename to excludeLogsDueToNoAccess()?

Reply:
OK will do